packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -61,674 +58,643 @@ <h1>Slider</h1>
     <hr />
     <br>
 
-    <h2>Default</h2>
+    <h2>Horizontal</h2>
     <section id="demo-default">
       <div class="iui-input-grid">
         <div class="iui-input-label">Volume</div>
-
-        <div class="iui-slider-component-container iui-slider-horizontal">
+        <div
+          class="iui-slider-container"
+          data-iui-orientation="horizontal"
+          style="--iui-slider-thumb-position: 50%;"
+        >
           <span class="iui-slider-min">
             <svg-sound-mute
-              class="iui-icon"
+              class="iui-svg-icon"
+              data-iui-icon-size="m"
               aria-hidden="true"
             ></svg-sound-mute>
           </span>
-          <div class="iui-slider-container iui-progress-slider">
-            <div class="iui-slider-rail"></div>
-            <div
-              class="iui-slider-track"
-              style="right: 50%;"
-            ></div>
+          <span class="iui-slider-max">
+            <svg-sound-loud
+              class="iui-svg-icon"
+              data-iui-icon-size="m"
+              aria-hidden="true"
+            ></svg-sound-loud>
+          </span>
```

Comment:
nit: Since on the React side, we are using the order of min-slider-max (https://github.com/iTwin/iTwinUI/pull/1606#discussion_r1346083116), maybe we can do the same here too.